src/main/java/com/klarna/hiverunner/HiveRunnerExtension.java

```diff
@@ -74,11 +74,15 @@ public void postProcessTestInstance(Object target, ExtensionContext extensionCon
     scriptsUnderTest = container.getScriptsUnderTest();
   }
 
-  private void setupConfig(Object target) {
-    Set<Field> fields = ReflectionUtils.getAllFields(target.getClass(),
-        Predicates.and(
-            withAnnotation(HiveRunnerSetup.class),
-            withType(HiveRunnerConfig.class)));
+    private void setupConfig(Object target) {
```

Comment:
Not sure this is really necessary but I can live with it ;)

Reply:
It is as reflections library is no using GUAVA Predicates anymore rather than java.util.function.Predicate